internal/surfer/gcloud/utils/field.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package utils provides utility functions for the gcloud generator.
+package utils
+
+import (
+	"github.com/googleapis/librarian/internal/sidekick/api"
+)
+
+// GetGcloudType maps a proto data type to its corresponding gcloud type.
+func GetGcloudType(t api.Typez) string {
```

Comment:
I don't think this should be a string, but hard to tell without seeing how it is used. It probably ought to be an enum that knows how to stringify.
We can do this for now and refactor later though.